sites/all/modules/hr/hr_datasets/hr_datasets.module

```diff
@@ -165,5 +179,31 @@ break;
   return $block;
 }
 
+/**
+* Function that gets all the tags from HDX.
+* @return array
+*/
+
+function hr_datasets_hdx_tags() {
+  $url = "https://data.hdx.rwlabs.org/api/3/action/tag_list";
+  $request = drupal_http_request($url);
+  if(!empty($request->data)) {
+    $result = drupal_json_decode($request->data);
+    return $result['result'];
+  }
+}
```

Comment:
This bit is identical in both functions, we should have a common way to read from HDX and use that utility function